package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "analytics-node",
```

Comment:
Do we have any preferences for package names for these things, @paddie?

This also comes down to distribution. I'm assuming we'll publish this fork to npm?
If so, can I suggest maybe making this a [scoped package](https://docs.npmjs.com/cli/v8/using-npm/scope)? This way, we could have the package `@dreamdata/analytics`, which would be quite sexy.

Reply:
I was looking at that as well but thought we just go with an unscooped module as the original was.

Reply:
As a user, I can only argue that I dislike unscoped packages. 😅 

I'll leave this to you and @paddie, just wanted to raise the suggestion. 👍

Reply:
I can create it as an scooped module instead if we want that under @dreamdata

Reply:
having it under `dreamdata` would be super nice @hanssonandreas , we might also want to look at the user-agent to make sure that it mentions `dreamdata` in some capacity.

Reply:
Talked to @paddie about it @wizhi and he like the idea of scooped so will change to that.

Reply:
Awesome, go sexy package names!